inclusive-design/eds.inclusivedesign.ca
# feat: update to Trivet 1.0.0 (resolves #13)

## Description

Update to the 1.0.0 release of [Trivet](https://github.com/fluid-project/trivet).

## Steps to test

1. run `npm start` to build and serve the site
2. It's currently just a bare Trivet instance, so links and UIO should work but the content isn't real

## Related issues

Resolves #13
